Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt

```diff
@@ -296,7 +298,7 @@ fun SnackImage(
     ) {
         AsyncImage(
             model = ImageRequest.Builder(LocalContext.current)
-                .data(imageUrl)
+                .data(imageRes)
                 .crossfade(true)
                 .build(),
             contentDescription = contentDescription,
```

Comment:
Should you consider removing the placeholder? You removed the jpg so maybe you also want to remove the placeholder parameter here.